### ISSUE TITLE
Unable to replicate cityscapes -> foggy cityscapes results

### ISSUE THREAD

**Issue by peterstratton** [BIT-DA/O2net#3]
I'm very interested in this work and have been attempting to reproduce the results. However, after following the instructions in the README, I have been unable to reproduce a mAP score of 47.2 after 50 epochs of training. Specifically, after pretraining a source model, then running domain adaptive training using the pretrained source model, I get a mAP score of 39.69. I trained using 1 GPU and a batch size of 2, the result of the experiment are shown below. 

However, I do not have the exact same system setup as described in the README. I'm using newer versions of Pytorch, Torchvision, and CUDA because my graphics card is too new and is not compatible with CUDA 9.2 anymore.

If someone could point me in the right direction to reproduce the results of this paper, that would be great!

IoU metric: bbox
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.21836
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.39669
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.20980
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.04972
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.22982
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.40898
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.18241
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.33332
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.35851
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.09898
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.35826
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.61922

**Comment by kxgong** [comment]
Thanks for your attention!

I see you used one GPU for training with batch size of 2.

You can try to use four GPUs for training (with a overall batch size of 8).

**Comment by peterstratton** [comment]
Ok sounds good, I'll give that a try and let you know the results!

**Comment by HaoooNPU** [comment]
hey, bro, did you reproduce the result?
